views/partials/analytics/experiment.hbs

```diff
@@ -1,3 +1,3 @@
 data-utm_term={{experimentBranch}}
 data-experiment={{experimentBranch}}
-data-utm_campaign="growthuserflow2"
+data-utm_campaign="growthuserflow3"
```

Comment:
Revised this ID for a clean break in the data

Reply:
Can we keep and append to `growthuserflow2`, e.g. `growthuserflow2a` so we can keep them sequentially in sync with our project docs?